CONTENT_ROUTING.md

```diff
@@ -0,0 +1,264 @@
+# ![](https://img.shields.io/badge/status-wip-orange.svg?style=flat-square) Content routing
+
```

Comment:
suggestion - start with a short recap or set of references to the status quo to orient new readers (landing on problems raises the questions "problems with what" if no background is given initially)